### DISCUSSION TITLE
Sliders during gameplay inside in middle have a "line".

### DISCUSSION THREAD

**Discussion by LunaticalFrost** [ppy/osu#19238] [Q&A] [answered]
This problem i have occured a long time ago, and i don't remember already, in what version that happened.
But there is a screenshot and logs attached to this post.
![osu_2022-07-19_16-15-13](https://user-images.githubusercontent.com/42748330/179765225-4107d9f7-0f53-414d-9911-5a3b2f047d98.jpg)
I writing on GitHub at first time, so i don't know what needed to exacly do to report issues correctly without missunderstanding and frustrating on side. But that "line" thing in slider is really distracting when i encounter to every time.
This problem reproduced in [this map](https://osu.ppy.sh/beatmapsets/1#osu/75)

[runtime.log](https://github.com/ppy/osu/files/9141538/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9141539/updater.log)
[database.log](https://github.com/ppy/osu/files/9141540/database.log)
[network.log](https://github.com/ppy/osu/files/9141541/network.log)
[performance.log](https://github.com/ppy/osu/files/9141542/performance.log)

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/3215.

**Reply by LunaticalFrost** [reply]
Thanks for reply
